Have to admit this is a little past me. Can't find a database?

santaclauslives opened the pull request:
Message below on Sinatra localhost:9292

ActiveRecord::NoDatabaseError at /
FATAL: database "postgres://localhost/debugging-challenge" does not exist
file: postgresql_adapter.rb location: rescue in connect line: 661

I ran bin/setup again and got the web page to pull up, but now have things I don't recognize in iTerm.

santaclauslives committed:
Have to admit this is a little past me. Can't find a database?

Message below on Sinatra localhost:9292

ActiveRecord::NoDatabaseError at /
FATAL: database "postgres://localhost/debugging-challenge" does not exist
file: postgresql_adapter.rb location: rescue in connect line: 661

santaclauslives commented:
Here is the errorr message I am getting. I doesn't really complete and give me the web-toy__d.... and so on at the command line again. I have to control-c to do anything else. 

[2016-02-08 13:25:47] INFO  WEBrick 1.3.1
[2016-02-08 13:25:47] INFO  ruby 2.3.0 (2015-12-25) [x86_64-darwin14]
[2016-02-08 13:25:47] INFO  WEBrick::HTTPServer#start: pid=30230 port=9292
WARN: tilt autoloading 'tilt/erb' in a non thread-safe way; explicit require 'tilt/erb' suggested.
D, [2016-02-08T13:26:04.210403 #30230] DEBUG -- :   Student Load (1.0ms)  SELECT "students".\* FROM "students"
D, [2016-02-08T13:26:04.304102 #30230] DEBUG -- :    (0.9ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 1]]
D, [2016-02-08T13:26:04.305489 #30230] DEBUG -- :    (0.4ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.312968 #30230] DEBUG -- :    (0.6ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 1]]
D, [2016-02-08T13:26:04.313954 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.316105 #30230] DEBUG -- :    (0.4ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 2]]
D, [2016-02-08T13:26:04.316791 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.318086 #30230] DEBUG -- :    (0.3ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 2]]
D, [2016-02-08T13:26:04.318792 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.320166 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 3]]
D, [2016-02-08T13:26:04.320873 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.322356 #30230] DEBUG -- :    (0.3ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 3]]
D, [2016-02-08T13:26:04.323071 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.324470 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 4]]
D, [2016-02-08T13:26:04.325223 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.326552 #30230] DEBUG -- :    (0.3ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 4]]
D, [2016-02-08T13:26:04.327320 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.328800 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 5]]
D, [2016-02-08T13:26:04.329623 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.331457 #30230] DEBUG -- :    (0.5ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 5]]
D, [2016-02-08T13:26:04.332147 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.333606 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 6]]
D, [2016-02-08T13:26:04.334494 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.335815 #30230] DEBUG -- :    (0.3ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 6]]
D, [2016-02-08T13:26:04.336596 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.337954 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 7]]
D, [2016-02-08T13:26:04.338703 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.340127 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 7]]
D, [2016-02-08T13:26:04.340791 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.342392 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 8]]
D, [2016-02-08T13:26:04.343151 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.344918 #30230] DEBUG -- :    (0.3ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 8]]
D, [2016-02-08T13:26:04.345734 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.347473 #30230] DEBUG -- :    (0.4ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 9]]
D, [2016-02-08T13:26:04.348299 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.349537 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 9]]
D, [2016-02-08T13:26:04.350250 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.351597 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 10]]
D, [2016-02-08T13:26:04.352373 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.353606 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 10]]
D, [2016-02-08T13:26:04.354316 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.355730 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 11]]
D, [2016-02-08T13:26:04.356417 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.357680 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 11]]
D, [2016-02-08T13:26:04.358374 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.359673 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 12]]
D, [2016-02-08T13:26:04.360355 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.361531 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 12]]
D, [2016-02-08T13:26:04.362242 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.363831 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 13]]
D, [2016-02-08T13:26:04.364566 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.365734 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 13]]
D, [2016-02-08T13:26:04.366448 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.367805 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 14]]
D, [2016-02-08T13:26:04.368532 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.369853 #30230] DEBUG -- :    (0.3ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 14]]
D, [2016-02-08T13:26:04.370562 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.371950 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 15]]
D, [2016-02-08T13:26:04.372647 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.373933 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 15]]
D, [2016-02-08T13:26:04.374648 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.376110 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 16]]
D, [2016-02-08T13:26:04.376920 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.378297 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 16]]
D, [2016-02-08T13:26:04.379001 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.380587 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 17]]
D, [2016-02-08T13:26:04.381650 #30230] DEBUG -- :    (0.4ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.383188 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 17]]
D, [2016-02-08T13:26:04.383926 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.385333 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 18]]
D, [2016-02-08T13:26:04.386030 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.387246 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 18]]
D, [2016-02-08T13:26:04.387960 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:26:04.388516 #30230] DEBUG -- :   Assignment Load (0.2ms)  SELECT "assignments".\* FROM "assignments"
::1 - - [08/Feb/2016:13:26:04 -0600] "GET / HTTP/1.1" 200 5886 0.3226
D, [2016-02-08T13:27:38.248832 #30230] DEBUG -- :   Student Load (0.3ms)  SELECT "students".\* FROM "students"
D, [2016-02-08T13:27:38.250747 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 1]]
D, [2016-02-08T13:27:38.251463 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.252760 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 1]]
D, [2016-02-08T13:27:38.253464 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.254986 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 2]]
D, [2016-02-08T13:27:38.256146 #30230] DEBUG -- :    (0.5ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.257599 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 2]]
D, [2016-02-08T13:27:38.258314 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.259916 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 3]]
D, [2016-02-08T13:27:38.260672 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.261890 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 3]]
D, [2016-02-08T13:27:38.262611 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.263920 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 4]]
D, [2016-02-08T13:27:38.264678 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.265873 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 4]]
D, [2016-02-08T13:27:38.266653 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.268012 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 5]]
D, [2016-02-08T13:27:38.268705 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.269894 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 5]]
D, [2016-02-08T13:27:38.270650 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.272526 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 6]]
D, [2016-02-08T13:27:38.273293 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.274563 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 6]]
D, [2016-02-08T13:27:38.275288 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.276647 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 7]]
D, [2016-02-08T13:27:38.277350 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.278574 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 7]]
D, [2016-02-08T13:27:38.279271 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.280598 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 8]]
D, [2016-02-08T13:27:38.281356 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.282655 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 8]]
D, [2016-02-08T13:27:38.283564 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.285253 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 9]]
D, [2016-02-08T13:27:38.286299 #30230] DEBUG -- :    (0.4ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.288014 #30230] DEBUG -- :    (0.4ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 9]]
D, [2016-02-08T13:27:38.289128 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.290567 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 10]]
D, [2016-02-08T13:27:38.291235 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.292406 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 10]]
D, [2016-02-08T13:27:38.293097 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.294543 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 11]]
D, [2016-02-08T13:27:38.295215 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.296442 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 11]]
D, [2016-02-08T13:27:38.297131 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.298425 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 12]]
D, [2016-02-08T13:27:38.299166 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.300364 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 12]]
D, [2016-02-08T13:27:38.301078 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.302448 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 13]]
D, [2016-02-08T13:27:38.303143 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.304320 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 13]]
D, [2016-02-08T13:27:38.305087 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.306905 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 14]]
D, [2016-02-08T13:27:38.307660 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.308886 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 14]]
D, [2016-02-08T13:27:38.309603 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.310905 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 15]]
D, [2016-02-08T13:27:38.311664 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.312860 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 15]]
D, [2016-02-08T13:27:38.313564 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.314860 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 16]]
D, [2016-02-08T13:27:38.315573 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.316756 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 16]]
D, [2016-02-08T13:27:38.317516 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.318852 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 17]]
D, [2016-02-08T13:27:38.319538 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.320772 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 17]]
D, [2016-02-08T13:27:38.321748 #30230] DEBUG -- :    (0.4ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.323213 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments" INNER JOIN "submissions" ON "assignments"."id" = "submissions"."assignment_id" WHERE "submissions"."student_id" = $1  [["student_id", 18]]
D, [2016-02-08T13:27:38.323857 #30230] DEBUG -- :    (0.2ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.325147 #30230] DEBUG -- :    (0.2ms)  SELECT SUM("grades"."score") FROM "grades" INNER JOIN "submissions" ON "grades"."submission_id" = "submissions"."id" WHERE "submissions"."student_id" = $1  [["student_id", 18]]
D, [2016-02-08T13:27:38.325968 #30230] DEBUG -- :    (0.3ms)  SELECT COUNT(_) FROM "assignments"
D, [2016-02-08T13:27:38.326512 #30230] DEBUG -- :   Assignment Load (0.2ms)  SELECT "assignments".\* FROM "assignments"
::1 - - [08/Feb/2016:13:27:38 -0600] "GET / HTTP/1.1" 200 5886 0.0843
^C[2016-02-08 13:36:53] INFO  going to shutdown ...
[2016-02-08 13:36:53] INFO  WEBrick::HTTPServer#start done.

sumeetjain commented:
Everything is working as it should, I think.

You can ignore the output of those logs until you reach a specific error.

What happens when you try each of the stories here: https://github.com/omahacodeschool/web-toy__debugging-challenge#assignment-details

The errors you see should be what you're supposed to fix.

But don't worry about what's being outputted in those logs until you reach an error.

santaclauslives committed:
I made what was probably a fatal mistake by backing out and starting over with bin/setup and rackup and so on. I get double everything now. Not really double, the scores for each student are no longer the same.

I was working on a "no defined method 'id' for 1:Fixnum" on line 50 of the student.rb file. The last thing I did was just remove the method ".id" from "assignment_object.id". (I don't know what "id" does, exactly, even after my Google search.)

Student Management Center

Welcome to the SMC. Below is a list of all students, linked to their profiles, and related information.

Student % Assignments Turned In Overall Grade Overall Grade, Adj. for Difficulty
Diana Mkrtchyan 10.0% 0
Heather Praisewater 40.0% 18
Nathaniel Ruiz  30.0% 18
Kyle Schmit 20.0% 11
Jen Homann  30.0% 8
Tony Plaza  0.0%  0
Diana Mkrtchyan 0.0%  0
Heather Praisewater 30.0% 17
Nathaniel Ruiz  40.0% 10
Kyle Schmit 0.0%  0
Jen Homann  30.0% 12
Tony Plaza  40.0% 27
All Assignments

Rock-Paper-Scissors
String Segementer
Prep Work
Sinatra Starter
Database Starter
Rock-Paper-Scissors
String Segementer
Prep Work
Sinatra Starter
Database Starter

santaclauslives commented:
Think I made a mistake and did another "git push" instead of stopping after the "git commit" stage.

santaclauslives commented:
I think I made a mistake. I wasn't getting what I thought I would get while trying to fix an error "no defined method "id" for 1:Fixnum did you mean 'i'?" on line 50 of student.rb. I did bin/setup again and rackup again and (and I knew this would happen about halfway through this process) and the web page became the following. I thought I could ignore and just go to the student pages and work on the errors there, but all I get is the Sinatra microphone. 

Student Management Center

Welcome to the SMC. Below is a list of all students, linked to their profiles, and related information.

Student % Assignments Turned In Overall Grade   Overall Grade, Adj. for Difficulty
Diana Mkrtchyan 10.0%   0  
Heather Praisewater 40.0%   18  
Nathaniel Ruiz  30.0%   18  
Kyle Schmit 20.0%   11  
Jen Homann  30.0%   8  
Tony Plaza  0.0%    0  
Diana Mkrtchyan 0.0%    0  
Heather Praisewater 30.0%   17  
Nathaniel Ruiz  40.0%   10  
Kyle Schmit 0.0%    0  
Jen Homann  30.0%   12  
Tony Plaza  40.0%   27  
All Assignments

Rock-Paper-Scissors
String Segementer
Prep Work
Sinatra Starter
Database Starter
Rock-Paper-Scissors
String Segementer
Prep Work
Sinatra Starter
Database Starter
